### ISSUE TITLE
asymmetric mini-notation explanation: slow down/repeat

### ISSUE THREAD

**Issue by jwaldmann** [tidalcycles/Tidal#648]
I'm not sure about the right place for the following remark. It's nothing critical. 

While preparing a presentation for a seminar, and looking up sources of documentation, I noticed that https://tidalcycles.org/index.php/Mini_notation_syntax uses asymmetric wording:
* `*` repeat a pattern
* `/` slow down a pattern

In the source code, it's `TPat_Fast` and `TPat_Slow`, translated to `fast` and `slow`, and ultimately, `_slow r p = _fast (1/r) p`, so it's fully symmetric. And I can indeed write `foo*2` as `foo/1%2`.

But then "repeat" should be "speed up"?

I agree that the common (beginners) usage is  `bd sn*2`
and there it looks like repetition. But that's related to the (mis)conception that `foo bar` means concatenation, while it actually is "play one cycle from each, in turn".

Well, tough times. Education is always about revelaing the truth only gradually...

**Comment by jwaldmann** [comment]
This text uses symmetric naming ("speeds up"/"slows down")
https://tidalcycles.org/index.php/Tutorial#Using_.2A_and_.2F_on_Groups

**Comment by yaxu** [comment]
Yes a tricky one! I still rather like 'density' and 'sparsity', but I have generally been standardising on 'speed up' and 'slow down'..

**Comment by ndr-brt** [comment]
Should this be moved to https://github.com/ndr-brt/tidal-doc ?

**Comment by yaxu** [comment]
Yes lets move all doc issues there. Shall I leave this one with you?

**Comment by ndr-brt** [comment]
Woa, I pasted the wrong link, sorry, I meant on the official one.

**Comment by yaxu** [comment]
yes thought so :)

**Comment by ndr-brt** [comment]
btw, I think we could move this too :)
